p2pd/main.go

```diff
@@ -8,12 +8,18 @@ import (
 
 	libp2p "github.com/libp2p/go-libp2p"
 	p2pd "github.com/libp2p/go-libp2p-daemon"
+	identify "github.com/libp2p/go-libp2p/p2p/protocol/identify"
 )
 
 func main() {
+	identify.ClientVersion = "p2pd"
```

Comment:
hmm should we get a version string in there? something like `fmt.Sprintf("/p2pd/%d", DaemonVersion)`

Reply:
Yeah, that's probably a good idea. But what is our current version?

Reply:
0.1!